### PR TITLE
Airflow

### PR DESCRIPTION
**Pull request by sbuss**
Initial airflow configuration; using a shared workdir directory for now. DataSF upload comes next

Diffbase = #161

### CODE REVIEW

File: schemaless/sources.py [hunk 1]
```diff
@@ -698,6 +705,8 @@ class PermitAddendaSummary(Source):
         'permit_number',
         'earliest_addenda_arrival'
     ]
+    DATA_SF = ""  # TODO: Not public
+    DATA_SF_DOWNLOAD = ""  # TODO: Not public
```
Comment (rajpara11) [inline schemaless/sources.py]: I think this is public?

File: schemaless/sources.py [hunk 2]
```diff
@@ -678,6 +684,7 @@ class OEWDPermits(DirectSource):
         'Target Permit Issuance Date': 'target_permit_issuance_date',
     }
     DATA_SF = "https://data.sfgov.org/dataset/Priority-Permits/336t-bzzm"
+    DATA_SF_DOWNLOAD = ""  # TODO: Not public
```
Comment (rajpara11) [inline schemaless/sources.py]: I'm following up with Jade about this but they've been busy/MIA, can we still do this if it is not public?

File: schemaless/sources.py [hunk 3]
```diff
@@ -382,6 +382,7 @@ class Planning(DirectSource):
         'blocklot': Concat('block', 'lot'),
     }
     DATA_SF = "https://data.sfgov.org/dataset/PPTS-Records_data/kgai-svwy"
+    DATA_SF_DOWNLOAD = "https://data.sfgov.org/api/views/kgai-svwy/rows.csv?accessType=DOWNLOAD"  # NOQA
```
Comment (rajpara11) [inline schemaless/sources.py]: Isn't this not public right now? How does this work but the other ones don't?

File: dag.py [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+    schedule_interval=timedelta(days=1),
+)
+
+task_create_schemaless = PythonOperator(
+    task_id='create_schemaless',
+    python_callable=create_schemaless.run,
+    op_kwargs={
+        'out_file': '{{ var.value.WORKDIR }}/schemaless.csv',
+        # 'no_download': 'True',
+        'planning_file': '{{ var.value.WORKDIR }}/../data/planning/planning-2020-03-11.csv.xz',  # NOQA
```
Comment (rajpara11) [inline dag.py]: How come this is explicitly pulled in?

File: dag.py [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+        'parcel_data_file': '{{ var.value.WORKDIR }}/../data/assessor/2020-02-18-parcels.csv.xz',  # NOQA
```
Comment (rajpara11) [inline dag.py]: Can we download this as well?

File: dag.py (hunk 4 above)
Comment (sbuss) [reply dag.py]: This dataset isn't public yet: https://data.sfgov.org/Housing-and-Buildings/SF-Planning-Permitting-Data/kncr-c6jw

File: dag.py (hunk 5 above)
Comment (sbuss) [reply dag.py]: Will fix in #169

File: schemaless/sources.py (hunk 1 above)
Comment (sbuss) [reply schemaless/sources.py]: Oh, is it this dataset? https://data.sfgov.org/Housing-and-Buildings/Department-of-Building-Inspection-Permit-Addenda-w/87xy-gk8d

File: schemaless/sources.py (hunk 3 above)
Comment (sbuss) [reply schemaless/sources.py]: This Source is no longer used, actually. I should delete it.

File: schemaless/sources.py (hunk 2 above)
Comment (sbuss) [reply schemaless/sources.py]: Technically, yes, but I'd have to rewrite the code to fetch datasets to authenticate & pull. Hoping they just make it public asap :)

File: schemaless/sources.py (hunk 1 above)
Comment (rajpara11) [reply schemaless/sources.py]: Yup

File: dag.py (hunk 4 above)
Comment (rajpara11) [reply dag.py]: Shouldn't we also have OEWD here as well then?

File: schemaless/sources.py (hunk 3 above)
Comment (sbuss) [reply schemaless/sources.py]: Edit: my mistake. I actually put the wrong URL here! The Source is right

File: schemaless/sources.py (hunk 1 above)
Comment (sbuss) [reply schemaless/sources.py]: fixed

File: dag.py (hunk 4 above)
Comment (sbuss) [reply dag.py]: Yep, fixed

File: schemaless/sources.py (hunk 3 above)
Comment (sbuss) [reply schemaless/sources.py]: fixed

File: schemaless/sources.py (hunk 2 above)
Comment (sbuss) [reply schemaless/sources.py]: Whatever solution we do pursue, I'd like to push it to a followup issue

Comment (rajpara11) [reply schemaless/sources.py]: sg, as talked about this won't be public so we'll either have to make it clear we're not using for DA and OCII projects or make ti work with the auth

**Pull request merged**